### ISSUE TITLE
Cannot tell the correct version of gnome terminal

### ISSUE THREAD

**Issue by harry830622** [metalelf0/gnome-terminal-colors#4]
On my 64-bit Arch Linux with Gnome desktop environment,
``` sh
$ gnome-terminal --version
GNOME Terminal 3.22.1 Using VTE version 0.46.1 +GNUTLS
```
But the gnome terminal version extracted by [tools.sh](https://github.com/metalelf0/gnome-terminal-colors/blob/master/src/tools.sh#L2) is `0.46.1 +GNUTLS` instead of `3.22.1`
Therefore, the profiles cannot be found and the installation will fail.

So could you fix the regex in [tools.sh](https://github.com/metalelf0/gnome-terminal-colors/blob/master/src/tools.sh#L2)?
Or maybe just echo the version and ask user if it's newer than 3.8?
I prefer latter because I think the format of `gnome-terminal --version` might be different in future again.

Thx a lot!!! :smile:

**Comment by aruhier** [comment]
Hi @harry830622,
It should be fixed since 0d74f5a.

Please tell me if it is not working for you, as [my solarized script](https://github.com/Anthony25/gnome-terminal-colors-solarized) would also be impacted.

**Comment by harry830622** [comment]
It works.
Thank you!!!
